OpenMTRDemo/Filters/CannyFilter.cs

```diff
@@ -20,8 +23,7 @@ public override void ApplyFilter(Mat image)
         }
 
         private void threshold_ValueChanged(object sender, System.EventArgs e)
-        {
-            int value;
+        {int value;
```

Comment:
This is rather odd (just the lack of  new line and indentation).

Reply:
That was unintentional.